Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -134,23 +122,10 @@ def findFile(workingDir, fileName, pathToResult=None):
 
   :return: list of paths leading to the file
   """
-
-  # In the case pathToResult is defined, we just have to check the path exists
-  if pathToResult:
-    path = os.path.join(workingDir, pathToResult, fileName)
-    if os.path.exists(path):
-      # We put the path in a list to be consistent
-      return S_OK([path])
-
-  # In the case pathToResult is not defined or not correct
-  # We have to search for the file in workingDir and can get multiple results
-  res = Subprocess().systemCall("find %s -name '%s'" % (workingDir, fileName), shell=True)
-  if not res['OK']:
-    return res
-  paths = res['Value'][1].splitlines()
-  if not paths:
-    return S_ERROR(errno.ENOENT, "Could not find %s in directory %s" % (fileName, workingDir))
-  return S_OK(paths)
+  path = os.path.join(workingDir, pathToResult, fileName)
+  if os.path.exists(path):
+    return S_OK(path)
```

Comment:
If you change the return type you must also adapt the documentation, and also fix getCondorLogFile